sklearn/linear_model/bayes.py

```diff
@@ -162,7 +162,8 @@ def fit(self, X, y):
         n_samples, n_features = X.shape
 
         # Initialization of the values of the parameters
-        alpha_ = 1. / np.var(y)
+        eps = np.spacing(1)
```

Comment:
You might want to add a small comment to explain why we add eps

Reply:
Thx for the suggestion. I added a comment, and additionally applied the same fix to `ARDRegression`

Reply:
I would prefer you use:

eps = np.finfo(np.float64).eps

as here you're not taking into account the dtype of X or y anyway

Reply:
done